tests/unit/theme-toggle.spec.ts

```
@@ -0,0 +1,283 @@
+      });
+    });
+
+    await page.goto('/settings');
+    await page.waitForLoadState('networkidle');
+
+    // Check button has emoji and text
+    const themeButton = page.locator('button').filter({ hasText: /Dark|Light/ });
+    const buttonText = await themeButton.textContent();
+    
+    // Should contain emoji and text
+    expect(buttonText).toMatch(/[☀️🌙]/);
```

Biome (2.1.2)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)